frontend/src/App.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
```

Comment:
Looks like the test name didn't get updated from project scaffolding although the test body did. No need to make changes here since the whole test is likely to be deleted once we have real functionality, but please keep test names in mind in the future.